libs/angular/src/lib/microzord-app/microzord-app.directive.ts

```diff
@@ -30,24 +40,30 @@ export class MicrozordAppDirective implements OnDestroy {
     private ngZone: NgZone,
     private errorHandler: ErrorHandler,
   ) {
-    this.application = this.name$.pipe(
+    const app$ = this.name$.pipe(
```

Comment:
I guess it could be made a bit more straightforward (without bunch of takeUntil and shares)
but i don't find any reasonable option yet